### DISCUSSION TITLE
Simulation of flow-boiling heat transfer

### DISCUSSION THREAD

**Discussion by jundonglin** [PMEAL/OpenPNM#2868] [Q&A] [answered]
Could  openpnm simulate flow-boiling heat transfer? It involves phase transitions, transients, three dimensions

**Comment by jgostick** [comment, marked as answer]
I think capturing the moving bubbles would be impossibly hard.

**Comment by jundonglin** [comment]
Could you tell  me roughly why it is so difficult to capture the moving bubbles?

**Reply by jgostick** [reply]
I don't think there is any software that can actually model moving bubbles at any appreciable scale...maybe 1 bubble in a small domain, but there is just too many physics to think about.  Basically, that is what experiments are for.
